src/components/Footer.js

```diff
@@ -3,30 +3,28 @@ import links from '../constants/links';
 
 const Footer = (props) => {
   return (
-    <footer id="footer">
-      <div className ="block-center">
-        <div className="social-icon-group">
-          <table>
-            <tr>
-              <td>
-                <a href={links.github} className="social-icon">
-                  <i className="fa fa-github fa-2x"></i>
-                </a>
-              </td>
-              <td>
-                <a href={links.email} className="social-icon">
-                  <i className="fa fa-envelope fa-2x"></i>
-                </a>
-              </td>
-            </tr>
-          </table>
+    <footer>
```

Comment:
Since this parent element is `footer` you could access child elements in the CSS by using a pattern like `footer .text` or to your desired level of specificity.